core/dbt/contracts/graph/model_config.py

```diff
@@ -407,6 +408,8 @@ class SavedQueryConfig(BaseConfig):
         default_factory=dict,
         metadata=MergeBehavior.Update.meta(),
     )
+    export_as: Optional[ExportDestinationType] = None
+    schema: Optional[str] = None
```

Comment:
are we dropping support for [alias](https://github.com/dbt-labs/dbt-core/pull/9065/files#diff-42f07118add483c2e34f2e1b77388819659667339ab74066353882b4adfed07eL738)?

Reply:
I intentionally left out alias because including it felt weird 🙃 If a saved query has multiple exports, I don't think we'd want them to all get aliased to to same thing. 

I didn't feel there was a great place to leave a comment about it (feels weird to comment in a class about why it doesn't have an attribute), so I put the informartion in the commit message
> We didn't add the `ExportConfig` `alias` property to the `SavedQueryConfig`. This
is because `alias` will always be specific to a single export, and thus it doesn't
make sense to allow defining it on the `SavedQueryConfig` to then apply to all
`Exports` belonging to the `SavedQuery`

Happy to add a comment to the class if that feels like the correct approach. Alternatively, if we do want to add alias we can, although I do worry that could cause other problems.

Reply:
Makes sense. Have we documented `alias` in user documentation yet? If so it might make sense to support some no-op backwards-compatible interface here given we are backporting this change.

Reply:
I will run through it with @graciegoheen / @jtcohen6. As of right now exports as a whole don't appear to be documented https://docs.getdbt.com/docs/build/saved-queries 🙃

Reply:
That's right! We closed out this docs issue until exports is fully ready to go https://github.com/dbt-labs/docs.getdbt.com/issues/4381

Relevant slack thread here -> https://dbt-labs.slack.com/archives/C05K4R7KZ5Z/p1699478482262759

Reply:
@QMalcolm Here's one new PR for saved queries:
https://github.com/dbt-labs/docs.getdbt.com/pull/4469